lib/forms/CoBrALab-Stress-Administration-Form/index.ts

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable perfectionist/sort-objects */
+
+const { defineInstrument } = await import('/runtime/v1/@opendatacapture/runtime-core/index.js');
+const { z } = await import('/runtime/v1/zod@3.23.x/index.js');
+
+function createDependentField<const T>(field: T, fn: (stressorType: string) => boolean) {
+  return {
+    kind: 'dynamic' as const,
+    deps: ['stressorType'] as const,
+    render: (data: { stressorType: string }) => {
+      if (fn(data.stressorType)) {
+        return field;
+      }
+      return null;
+    }
+  };
+}
+
+export default defineInstrument({
+  kind: 'FORM',
+  language: 'en',
+  tags: ['Stress', 'Tail suspension', 'Restraint', 'Electric shock'],
+  internal: {
+    edition: 1,
+    name: 'STRESS_ADMINISTRATION_FORM'
+  },
+  content: {
+    miceNumber: {
+      kind: "number",
+      variant: "slider",
+      label: "Number of mice",
+      max: 10,
+      min: 1
+    },
```

Comment:
What's the relevance here?

Reply:
there can be multiple mice taking the test at once (ex: multiple mice in the foot shock chamber)

Reply:
ok